itou/templates/siaes/job_description_list.html

```diff
@@ -1,204 +1,143 @@
+                    <h1 class="s-title-01__title h1">
+                        <strong>Métiers et recrutements</strong>
+                    </h1>
+                    <h2 class="h2">{{ siae.kind }} {{ siae.display_name }}</h2>
+                </div>
+            </div>
+        </div>
+    </section>
 {% endblock %}
 
+{% block content_container_css_classes %}s-section pt-0 mt-0{% endblock %}
 {% block content %}
-
-    <h1 class="mb-0">{{ siae.kind }} {{ siae.display_name }}</h1>
-    <h2 class="text-muted">Gérer les métiers et recrutements</h2>
-
-    <div class="shadow rounded p-4">
-        <div class="row">
-            <div class="col">
-                <h6>
-                    Le{{ job_pager.paginator.count|pluralize }} métier{{ job_pager.paginator.count|pluralize }} exercé{{ job_pager.paginator.count|pluralize }} dans votre structure
-                </h6>
-            </div>
-            <div class="col text-right">
-                <a class="btn btn-primary btn-sm" href="{% url "siaes_views:edit_job_description" %}?page={{ page|default:1 }}" aria-label="Créer une nouvelle fiche de poste">Créer une nouvelle fiche de poste</a>
+    <div class="c-box">
+        <div class="d-flex flex-column flex-md-row align-items-md-center justify-content-md-between">
+            <h3 class="h4 mb-0">
+                Métier{{ job_pager.paginator.count|pluralize }} exercé{{ job_pager.paginator.count|pluralize }}
+            </h3>
+            <div class="flex-column flex-md-row btn-group btn-group-sm btn-group-action" role="group" aria-label="Actions sur les candidatures">
+                {% if job_pager %}
+                    <form method="post" action="?page={{ page|default:1 }}" id="block_job_applications_form">
+                        {% csrf_token %}
+                        <input class="sr-only invisible" type="checkbox" name="block_job_applications" id="block_job_applications" {% if siae.block_job_applications %}checked{% endif %} />
+                        <label for="block_job_applications" class="d-block w-100 btn btn-sm btn-secondary btn-ico justify-content-center mt-3 mt-md-0 mb-0">
+                            {% if siae.block_job_applications %}
+                                <i class="ri-lock-unlock-line ri-lg"></i>
+                            {% else %}
+                                <i class="ri-lock-line ri-lg"></i>
+                            {% endif %}
+                            {% if siae.block_job_applications %}
+                                <span>Débloquer l'envoi de candidatures</span>
+                            {% else %}
+                                <span>Bloquer l'envoi de candidatures</span>
+                            {% endif %}
+                        </label>
+                    </form>
+                {% endif %}
+                <a class="btn btn-primary btn-ico justify-content-center mt-3 mt-md-0" href="{% url "siaes_views:edit_job_description" %}?page={{ page|default:1 }}">
```

Comment:
Le paramètre `page` ne semble pas utilisé par la vue `edit_job_description` mais seulement pour `job_description_list`
```suggestion
                <a class="btn btn-primary btn-ico justify-content-center mt-3 mt-md-0" href="{% url "siaes_views:edit_job_description" %}">
```